CONTRIBUTING.md

```diff
@@ -36,6 +36,22 @@ Once approved, we will merge your changes.
 
 As soon as your pull request is merged, a new build automatically occurs and your changes publish to [https://docs.temporal.io](https://docs.temporal.io).
 
+### Prettier
```

Comment:
AFAIK - You don't need to do this if you are a member of the core Temporal group, as you can open a pr for a branch off of origin.

You only need to do this if you fork the repo and are opening a pr from an upstream remote.